src/index.css

```diff
@@ -0,0 +1,169 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.App {
+  background-color: rgb(81, 130, 221);
+  width: 90%;
+  margin: 1rem auto;
+  border-radius: 15px;
+  box-shadow: 10px 10px 5px 0 rgba(0, 0, 0, 0.75);
+  -webkit-box-shadow: 10px 10px 5px 0 rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: 10px 10px 5px 0 rgba(0, 0, 0, 0.75);
+}
+
+.header-container {
+  display: flex;
+  padding: 1rem;
+  margin-bottom: 1rem;
+}
+
+.angle-Left {
+  display: flex;
+  justify-content: flex-start;
+  width: 80%;
+}
+
+.left,
+.mic,
+.ger,
+h1,
+h3 {
+  color: #e9e2e2;
+}
+
+.cont {
+  width: 20%;
+  display: flex;
+  justify-content: right;
+  gap: 1rem;
+}
+
+.hero {
+  background-color: rgb(65, 102, 174);
+  width: 90%;
+  height: 35vh;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+  padding: 1rem;
+}
+
+h1,
+h3 {
+  text-align: center;
+  margin: 1rem;
+}
+
+span {
+  padding: 1rem;
+}
+
+.search {
+  display: block;
+  width: 50%;
+  padding: 0.5rem;
+  margin: 1rem auto;
+}
+
+.cards {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 5px;
+}
+
+.card {
+  position: relative;
+  border-radius: 8px;
+  overflow: hidden;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  background-color: #fff;
+  width: 45%;
+  transition: transform 0.2s;
+}
+
+.blue {
+  background-color: rgb(65, 102, 174);
+}
+
+.light-blue {
+  background-color: rgb(84, 134, 225);
+}
+
+.card:hover {
+  transform: translateY(-5px);
+}
+
+.card-image {
+  max-width: 30%;
+  height: auto;
+}
+
+.card-content {
+  padding: 16px;
+}
+
+.card-title {
+  font-size: 1.5rem;
+  margin: 0;
+  color: #e9e2e2;
+}
+
+.card-description {
+  margin-top: 8px;
+  font-size: 1rem;
+  color: #e9e2e2;
+}
+
+.details-card {
+  position: relative;
+}
+
+.details-card ul {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  justify-content: space-around;
+  margin: 0 auto;
+}
+
+.details-card ul li {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 95%;
+  padding: 5px;
+  margin: 5px;
+}
```

Comment:
- [ ] kindly make your date in human-readable form and also make sure the text doesn't stick out of its container
![image](https://github.com/MostafaShahat2021/reactCapstoneProject/assets/66097371/14d7d42b-6b05-4a05-9afc-1df504d771ef)

Reply:
Thanks I'll fix it 👍